.github/workflows/ci.yml

```diff
@@ -13,9 +13,32 @@ on:
 
 jobs:
 
+  fast-checks:
+    name: Fast
+    uses: ./.github/workflows/checks.yml
+    strategy:
+      matrix:
+        working-directory:
+          - pytest-backend
+          - pytest-extension
+          - pytest-slc
+    with:
+      working-directory: ${{ matrix.working-directory }}
+
   Verify:
-    name: Tests
-    runs-on: ubuntu-20.04
+    name: Slow Tests (${{ matrix.working-directory }})
+    runs-on: ubuntu-24.04
+    needs: [ fast-checks ]
+    strategy:
+      matrix:
+        working-directory:
+          - pytest-backend
+          - pytest-extension
+          - pytest-slc
+      max-parallel: 1
```

Comment:
:question: Is it known whether the SAAS resource can handle multiple connection and that these tests are separate and could not interfere with each other?

Reply:
Each test run starts its own SaaS DB